src/engine/Graphics/Context/draw-image-command.ts

```diff
@@ -0,0 +1,141 @@
+import { Matrix } from '../../Math/matrix';
```

Comment:
Wonder if these classes/modules could benefit from comments while it's still fresh in your mind.